modules/virt-monitoring-upgrade-status.adoc

```diff
@@ -35,7 +35,7 @@ $ oc get csv
 [source,terminal,subs="attributes+"]
 ----
 VERSION  REPLACES                                        PHASE
-{HCOVersion}    kubevirt-hyperconverged-operator.v{LastHCOVersion}         Installing
+2.5.0    kubevirt-hyperconverged-operator.v{LastHCOVersion}         Installing
```

Comment:
This is getting more complex,
2.4.4 got released after 2.5.0 so now the valid upgrade paths are:
- 2.4.3 -> 2.5.0 -> 2.5.1 -> 2.5.2 -> 2.5.3
- 2.4.4 -> 2.5.2 -> 2.5.3